plugins/azure_sentinel/plugin.spec.yaml

```diff
@@ -0,0 +1,808 @@
+      type: "[]string"
+    timeGenerated:
+      description: The time the alert was generated
+      title: Title 
+      type: date
+    type:
+      description: Azure resource type
+      title: Title 
+      type: string
+    vendorName:
+      description: The name of the vendor that raised the alert
+      title: Vendor Name
+      type: string
+  Alert:
+    properties:
+      title: Properties
+      description: Alert's properties
+      type: AlertProperties
+  # Incident Types
+  IncidentOwnerInfo:
+    assignedTo:
+      description: The name of the user the incident is assigned to
+      title: Assigned To
+      type: string
+    email:
+      description: The mail of the user the incident is assigned to
+      title: Email
+      type: string
+    objectId:
+      description: The object id of the user the incident is assigned to
+      title: Object ID
+      type: string
+    userPrincipalName:
+      description: The user principal name of the user the incident is assigned to
+      title: User Principal Name
+      type: string
+  IncidentLabel:
+    labelName:
+      title: Label Name 
+      description: The name of the label
+      type: string
+    labelType:
+      title: The type of label
+      description: Label Type 
+      type: string
+      example: System
+  IncidentAdditionalData:
+    alertProductNames:
+      description: List of product names of alerts in the incident
+      title: Alert Product Names 
+      type: "[]string"
+    alertsCount:
+      description: The number of alerts in the incident
+      title: Alert's Count 
+      type: integer
+    bookmarksCount:
+      description: The number of bookmarks in the incident
+      title: Bookmarks Count 
+      type: integer
+    commentsCount:
+      description: The number of comments in the incident
+      title: Comments Count 
+      type: integer
+    tactics:
+      description: The tactics associated with incident
+      title: Tactics 
+      type: "[]string"
+  IncidentProperties:
+    additionalData:
+      description: Additional data on the incident
+      title: Additional Data 
+      type: IncidentAdditionalData
+    classification:
+      description: The reason the incident was closed
+      title: Classification 
+      type: string
+      example: BenignPositive
+    classificationComment:
+      description: Describes the reason the incident was closed
+      title: Classification Comment
+      type: string
+    classificationReason:
+      description: The classification reason the incident was closed with
+      title: Classification Reason
+      type: string
+      example: InaccurateData 
+    createdTimeUtc:
+      description: The time the incident was created
+      title: Created Time UTC
+      type: date
+    description:
+      description: The description of the incident
+      title: Description 
+      type: string
+    etag:
+      description: Etag of the azure resource
+      title: Etag
+      type: string
```

Comment:
Vendor API returns etag in a string. Kindly normalize it. Example: Get incident action is returning:  "etag": "\"bd006283-0000-0100-0000-6230e4c90000\"", instead of  "etag": "bd006283-0000-0100-0000-6230e4c90000",